dev/TreeView/TreeViewItem.xaml

```diff
@@ -341,7 +341,13 @@
                                         </Style>
                                     </Grid.Resources>
 
-                                    <CheckBox x:Name="MultiSelectCheckBox" Width="32" VerticalAlignment="Stretch" Style="{StaticResource TreeViewMultiSelectCheckBox}" Visibility="Collapsed" IsTabStop="False"/>
+                                    <CheckBox x:Name="MultiSelectCheckBox"
+                                              Width="32"
+                                              VerticalAlignment="Stretch"
+                                              Style="{StaticResource TreeViewMultiSelectCheckBox}"
+                                              Visibility="Collapsed"
+                                              IsTabStop="False"
+                                              AutomationProperties.AccessibilityView="Raw"/>
```

Comment:
Do we also need to apply this to TreeViewItem_rs1.xaml? Or is there an opportunity to merge those two using conditional XAML? @licanhua?

Reply:
I've updated rs1 style as well. 

---
In reply to: [249972142](https://github.com/Microsoft/microsoft-ui-xaml/pull/210#discussion_r249972142) [](ancestors = 249972142)